pycbc/events/events.py

```diff
@@ -136,7 +136,9 @@ def findchirp_cluster_over_window(times, values, window_length):
     indices: Array
         The reduced list of indices of the SNR values
     """
-    assert window_length > 0, 'Clustering window length is not positive'
+    #Commented out the below assert statement to allow for no clustering
+    #in the coherent matched_filter codes
+    #assert window_length > 0, 'Clustering window length is not positive'
```

Comment:
Can you revert this change? (Mixing in a change here is going to be problematic with other development work to move to python 3). For now just add a catch in the coherent code to do nothing if window_length is 0.

Reply:
I reverted this without any other changes because I couldn't remember where it caused problems, and it ran fine, so I think we can just revert it and leave it at that. (Though I assume I got rid of it at some point for a reason so this may cause problems in the future.)